_posts/0000-01-02-aly-jpg.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![meandjoe](https://user-images.githubusercontent.com/65631821/82388654-1a4ea900-9a08-11ea-82b0-9cd093fdfa41.JPG)

Reply:
Add Benny Drama and Mercedes